lib/themes-panel.coffee

```diff
@@ -147,6 +153,43 @@ class ThemesPanel extends View
     @activeUiTheme = @getActiveUiTheme()
     @activeSyntaxTheme = @getActiveSyntaxTheme()
     @populateThemeMenus()
+    @toggleActiveThemeButtons()
+    @handleActiveThemeButtons()
+
+  handleActiveThemeButtons: ->
```

Comment:
Super minor, but maybe `handleEvents` is a better name here? Reading `handle...buttons` makes it feel like it is missing a word.